baseapp-auth/baseapp_devices/models.py

```diff
@@ -0,0 +1,29 @@
+import swapper
+from django.conf import settings
+from django.db import models
+
+
+class BaseUserDevice(models.Model):
+    class Meta:
+        abstract = True
+
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        related_name="devices",
+        on_delete=models.CASCADE,
+    )
+    ip_address = models.CharField(max_length=255)
```

Comment:
What if we change to [GenericIPAddressField](https://docs.djangoproject.com/en/5.1/ref/models/fields/#genericipaddressfield) instead?